pyproject.toml

```diff
@@ -39,6 +39,7 @@ dev = [
     "pytest-cov~=4.1.0",
     "pytest-asyncio~=0.23.0",
     "python-dotenv~=1.0.0",
+    "ipykernel~=6.29",
```

Comment:
Add whole `jupyter` dependency.

Reply:
it seems that it is not required, ipykernel is enough to run in VSCode

Reply:
removed the dependency